packages/sdk-middleware-auth/src/existing-token.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+import type {
+  Middleware,
+  MiddlewareRequest,
+  MiddlewareResponse,
+  ExistingTokenMiddlewareOptions,
+  Next,
+} from '../../../types/sdk'
+
+export default function createAuthMiddlewareForExistingToken(
+  options: ExistingTokenMiddlewareOptions | string
+): Middleware {
+  return (next: Next) => (
+    request: MiddlewareRequest,
+    response: MiddlewareResponse
+  ) => {
+    const config = options
+      ? {
+          tokenType:
+            typeof options === 'object' && options !== null && options.tokenType
+              ? options.tokenType
+              : 'Bearer',
+          token: typeof options === 'string' ? options : options.token,
+          force: options.force !== undefined ? options.force : true,
+        }
+      : null
```

Comment:
Maybe we can make use of Lodash's `default` function here. That will also allow us to define the default options as a variable that we can refer to in order to find out what the default options are.

Reply:
Maybe something like this:
```js
const defaultOptions = {
  tokenType: 'Bearer',
  force: false,
};
if (typeof options === 'string') {
  options = { ...defaultOptions, token: options }
} else if (typeof options === 'object') {
  options = default(defaultOptions, options)
}
```